metricbeat/module/system/diskio/diskio.go

```diff
@@ -107,6 +107,7 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 			},
 			"io": common.MapStr{
 				"time": counters.IoTime,
+				"ops":  counters.IopsInProgress,
```

Comment:
It looks like this metric is exclusive to linux. We're trying to move away from globally reporting metrics that are exclusive to one or two OSes, as it means that unsupported OSes will report an unhelpful `0`. We'll want to insert this into the MapStr later with a `runtime.GOOS == "linux"` guard.